ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-commands.sh

```diff
@@ -4,6 +4,17 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
```

Comment:
if the mirror registry is behind a proxy, I think we should move the export proxy block to a place as possible as close to the oc mirror command, e.g: https://github.com/openshift/release/pull/56465/files#diff-bdea30af18dd50cc47a8f900a78e8b74d7a1f09860173647789b129acf41dc8dR118

Once export proxy here, all the traffic will go through it, even interact with build farm cluster.

Reply:
I'm fine with this request, I'll do it right now.